notebooks/notebooks/paper/katago-new-ckpt-evals.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        # Remove last entry from legend
</pre><p>I think the comment here could be a bit clearer — we're moving the legend adv. training line out to a separate legend box in order to save space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>